main.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+	"os/signal"
+
+	v1alpha1agg "github.com/talos-systems/os-engine/internal/aggregator/v1alpha1"
+	"github.com/talos-systems/os-engine/internal/loader"
+	"github.com/talos-systems/os-engine/pkg/sink"
+	"github.com/talos-systems/os-engine/pkg/source"
+)
+
+type Options struct {
+	Address string
+	Plugins string
+}
+
+var options Options
+
+func init() {
+	flag.StringVar(&options.Plugins, "plugins-dir", "/lib/engine", "the directory to load plugins from")
+	flag.StringVar(&options.Address, "grpc-address", "", "the gRPC address to connect to")
```

Comment:
This should be `runtime-address`. We should enforce that only one runtime is used at a time.